test/e2e_node/pod_conditions_test.go

```diff
@@ -134,6 +134,62 @@ func runPodFailingConditionsTest(f *framework.Framework, hasInitContainers, chec
 		// Verify PodReady is not set (since sandboxcreation is blocked)
 		_, err = getTransitionTimeForPodConditionWithStatus(p, v1.PodReady, false)
 		framework.ExpectNoError(err)
+
+		ginkgo.By("checking pod condition for a pod when volumes is created")
+
+		configmap := v1.ConfigMap{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "cm-that-unblock-pod-condition",
+			},
+			Data: map[string]string{
+				"key": "value",
+			},
+			BinaryData: map[string][]byte{
+				"binaryKey": []byte("value"),
+			},
+		}
+
+		_, err = f.ClientSet.CoreV1().ConfigMaps("default").Create(ctx, &configmap, metav1.CreateOptions{})
+		framework.ExpectNoError(err)
+
+		_, err = f.ClientSet.CoreV1().ConfigMaps("default").Get(ctx, "cm-does-not-exist", metav1.GetOptions{})
```

Comment:
This probably should be cm-that-unblock-pod-condition rather than cm-does-not-exist.